faust-src/volumedsp.dsp

```diff
@@ -7,6 +7,8 @@ declare description "Volume Control Faust Plugin for JackTrip, based on Faust ex
 
 import("stdfaust.lib");
 mute = checkbox("[1] Mute");
-gain = hslider("[0] Volume", 0, -40, 0, 0.1) : ba.db2linear : si.smoo : _;
-gainVMute = _ * gain, 0 : select2(mute) : _;
-process = _ <: vgroup("Volume Control", _ : gainVMute);
+gain(v) = v : ba.db2linear : si.smoo : _;
+gainVMute(v) = _ * gain(v), 0 : select2(mute) : _;
+zeroCutoff(v) = _ * gain(v), 0 : select2(v == -40) : _;
```

Comment:
I think `gain()` is being called in twice here in the chain It so it would still work, though the scaling would be different (i.e. setting the slider 50% has the effect of using 25% volume).

![Screen Shot 2022-10-06 at 11 35 08 AM](https://user-images.githubusercontent.com/26987971/194392241-45df0e77-d25c-4fcd-81ab-d7f707774ca5.png)
![Screen Shot 2022-10-06 at 11 35 14 AM](https://user-images.githubusercontent.com/26987971/194392246-583ea403-a6ef-4109-8032-1bbeaa24a785.png)

I think this line should be:
`zeroCutoff(v) = _ , 0 : select2(v == -40) : _;`

Reply:
Oh yeah! This explains that thing I asked you about in slack.